oracle_en/index.js

```diff
@@ -1,87 +1,99 @@
 const contractSource = `
-contract Oracles =
+contract Source =
   record state = {  									
     source_oracle : map(address, oracle(string, string)),
     id_query : map(address, oracle_query(string, string)),
     question_answer : map(string, string)}
+
   stateful entrypoint init() : state =  						 
     { source_oracle = {},
       id_query = {}, 
       question_answer  = {} }
-  stateful entrypoint registerOracle(      						
+
+  stateful entrypoint register_oracle(      						
         qfee : int,     						 //Minimum payment fee
         rttl : int) : oracle(string, string) =   //oracle expiration time blocks
     let register: oracle(string, string) =  Oracle.register(Contract.address, qfee, RelativeTTL(rttl))
     put(state{ source_oracle[Contract.address] = register })
     register
+
   entrypoint get_oracle(): oracle(string, string) =  					
     switch(Map.lookup(Contract.address, state.source_oracle))
       None    => abort("Not registered")
       Some(x) => x
-  stateful entrypoint extendOracle(  							
+
+  stateful entrypoint extend_oracle(  							
                                 o   : oracle(string, string),	//oracle address
                                 ttl : int) : unit =		//oracle expiration time blocks 
     Oracle.extend(o, RelativeTTL(ttl))
-  payable stateful entrypoint quest_answer(quest : string, answ : string) : bool = 
-    let val = Call.value
-    if(val > 0)
-      false
-    else
-      put(state{question_answer[quest] = answ })
-      true
-  entrypoint getQuestion(  								
+
+  stateful entrypoint quest_answer(quest : string, answ : string) : bool = 
+    put(state{question_answer[quest] = answ })
+    true
```

Comment:
why is there the need to return true?

Reply:
The bolean is used for validation purposes

Reply:
why is that validation needed, either it return or it aborts, the boolean is not needed then